app/models/book.rb

```diff
@@ -131,55 +129,31 @@ def price
   end
 
   def as_json(options={})
-    super(
-      methods: [
-        :genre_name,
-        :subcategory_name,
-        :language_name,
-        :country_name,
-        :levels_name,
-        :publisher_name,
-        :authors_name,
-        :update_status,
-        :donated?,
-        :updated_date,
-        :url
-      ]
-    )
+    options[:methods] = [:subcategory_name, :update_status, :donated?, :updated_date, :url]
+    super(options)
+  end
+
+  def as_indexed_json(options={})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `as_indexed_json(*)` if you want the method to accept any arguments but don't care about them.<br>Inconsistent indentation detected.<br>Surrounding space missing in default value assignment.